parts/linux/cloud-init/artifacts/labels.sh

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env bash
+
+# Update Labels for Kubernetes nodes
+
+set -euo pipefail
+
+kubectl label --overwrite nodes $NODE_NAME $KUBELET_NODE_LABELS
```

Comment:
how confident are you that these will be available here.

Reply:
$NODE_NAME was my template variable. I will update it to an actual template <Node Name>. I was unable to easily find where to get the current node name from.
```suggestion
kubectl label --overwrite nodes <Node Name> $KUBELET_NODE_LABELS
```

I'm not sure if $KUBELET_NODE_LABELS is accessible or not.

Reply:
Just found out that $NODE_NAME is set to $(hostname) in cse_config,sh which Wei and I expect is the node name; although, I have not confirmed that. I expect that is accessible here, but I am not sure about $KUBELET_NODE_LABELS

Reply:
it should be correct. I checked on an AKS node. It has to be $(hostname) or $HOSTNAME but can't be $hostname because the env variable is uppercased

![image](https://user-images.githubusercontent.com/6925089/101232620-6d681080-3667-11eb-9ce5-7dbba91ddfd1.png)

Reply:
node labels should be accessible via the template func `{{ GetAgentKubernetesLabels }}`

Reply:
Thanks for the comment @eriksywu; however, I didn't end up using {{ GetAgentKubernetesLabels }}.

If I remember right I was trying to use it within the .service or .sh files and it didn't get converted correctly, but I might be misremembering, or did something wrong. I was able to get the labels by using the same env file that kubectl used.